openwisp_controller/config/templates/admin/config/change_form.html

```diff
@@ -1,6 +1,15 @@
 {% extends "admin/change_form.html" %}
 {% load i18n admin_modify %}
 
+{% block extrastyle %}
+{{ block.super }}
+{% if CONFIG_BACKEND_FIELD_SHOWN is False %}
```

Comment:
I don't see the code which passes this setting to the template, have you tested this?
It should not work with the current implementation because ``CONFIG_BACKEND_FIELD_SHOWN`` will be undefined and hence ``None``. It must be passed in the context (see https://github.com/openwisp/openwisp-controller/blob/master/openwisp_controller/config/admin.py#L86, ensure to add this to the context only if the model is not `Vpn`).

Reply:
Ok, this is getting passed to the template
https://github.com/openwisp/openwisp-controller/blob/5cfd578b0e5eed8818eb7f7849b8e23e68682095/openwisp_controller/config/admin.py#L101
I will change this to work only when model is not Vpn

Reply:
Done, I have tested the following:
- [x] Config editor does not break in `Device` and `Template`
- [x] Backend field hidden when `CONFIG_BACKEND_FIELD_SHOWN` is false in `Device` and `Template` but `Vpn` unaffected
- [x] Backend field hidden from list and filter in `Device` and `Template` but `Vpn` unaffected